docs/src/main/mdoc/unsafe_module.md

```diff
@@ -40,3 +40,57 @@ import monocle.macros.GenLens
 ```
 
 In this example the age is reset to `0` which invalidates the original predicate of `age >= 18`. More formally `UnsafeSelect` can invalidate the `roundTripOtherWayLaw` law.
+
+## MapTraversal
+
+`MapTraversal` provides an `Iso` `allKeyValues` between `Map[K,V]` and `List([K, V])` and Traversal `mapKVTraversal` of `Map[K, V]` to `(K, V)`. They are useful for traversing and modifying the entries of a map.
+
+Both of them are unsafe because of key collision and the unorderness of map entries. As a rule of thumb, laws regarding modifying `(K, V)` and then getting back a `List[(K, V)]` could be broken, while laws regarding modifying `(K, V)` and then getting back a `Map[K, V]` could still hold. For example,
+
+```scala mdoc:silent
+import monocle.unsafe.MapTraversal.allKeyValues
+
+val list = List((1, "foo"), (1, "bar"))
+val list2 = (allKeyValues[Int, String].get _ compose allKeyValues[Int, String].reverseGet)(list)
+// list2: List[(Int, String)] = List((1,bar))
+```
+
+`list` and `list2` does not equal here due to key collision. Even if there is no key collision, the output list is still not guaranteed to be the original list as the order of map entries maybe unspecified.
+
+On the other hand, the composition of `reverseGet` and `get` is identity.
+
+```scala mdoc:silent
+import monocle.unsafe.MapTraversal.allKeyValues
+
+val map = Map(1 -> "foo", 1 -> "bar")
+// map: scala.collection.immutable.Map[Int,String] = Map(1 -> foo, 2 -> bar)
```

Comment:
did you manually add the result? I think this should be produced by mdoc no?

Reply:
I am not familiar with mdoc. The current form of the results are obtained from scala repl and some manual work.

Reply:
Yes, I don't think you need any manual work. Locally, you can run `sbt makeMicrosite` to build the site and then use jekyll to see a local version as in: https://47degrees.github.io/sbt-microsites/docs/build-the-microsite/

But I think it is enough to remove the repl comment.

Reply:
Thanks for the explanation. I removed the comments.